test/unit/app/managers/test_HDCAManager.py

```diff
@@ -62,7 +64,7 @@ def testable_url_for(*a, **k):
 class HDCASerializerTestCase(HDCATestCase):
     def set_up_managers(self):
         super().set_up_managers()
-        self.hdca_serializer = hdcas.HDCASerializer(self.app)
+        self.hdca_serializer = hdcas.HDCASerializer(cast(StructuredApp, self.app))
```

Comment:
I'm wondering if it would be easier to make the cast in `test/unit/app/managers/base.py::BaseTestCase.set_up_mocks()` instead?

Reply:
that turns out to case other issues